responsibleai/responsibleai/managers/causal_manager.py

```diff
@@ -328,7 +329,25 @@ def get(self):
         return self._results
 
     def list(self):
-        pass
+        """List information about the CausalManager.
+
+        :return: A dictionary of properties.
+        :rtype: Dict
+        """
+        props = {ListProperties.MANAGER_TYPE: self.name}
+        causal_props_list = []
+        for result in self._results:
+            causal_config_dict = result.config.get_config_as_dict()
+            causal_config_dict['global_effects_computed'] = \
```

Comment:
I think these should be constants? either in this file or in some global constants file.

Reply:
Done